src/deploy.js

```diff
@@ -1,15 +1,85 @@
-import { spawn } from './utils';
+import AWS from 'aws-sdk';
```

Comment:
Let's stick to either `aws` or `Aws`.

Reply:
Changed to  `aws`

Reply:
I just pushed the latest changes. Please check again